servicecontrol/servicecontrol-instances/remotes.md

```diff
@@ -157,9 +157,9 @@ class primaryA,primaryB,crossRegionPrimary ServiceControlPrimary
 class auditA,auditB ServiceControlRemote
 ```
 
-In this deployment, each region has a full ServiceControl installation with a primary instance and an Audit instance. Each region can be managed and controlled via a dedicated ServicePulse.
+In this deployment, each region has a full ServiceControl installation with a primary Error instance and an Audit instance. Each region can be managed and controlled via a dedicated ServicePulse.
 
-A new cross-region primary instance is added to allow ServiceInsight to show messages from both regions. This cross-region instance includes each region-specific primary instance as a remote allowing it to query messages from both. The cross-region instance should disable error queue management by configuring the [error queue](/servicecontrol/creating-config-file.md#transport-servicebuserrorqueue) with the value `!disable`.
+A new cross-region primary instance is added to allow ServiceInsight to show messages from both regions. This cross-region instance includes each region-specific primary Error instance as a remote allowing it to query messages from both. The cross-region instance should disable error queue management by configuring the [error queue](/servicecontrol/creating-config-file.md#transport-servicebuserrorqueue) with the value `!disable`.
```

Comment:
Is there are reason for keeping "primary" in these lines?

Reply:
The terms primary and remote have specific meanings in this document. I believe you can have remote Error instances and primary Error instances. Audit instances can never be primary and are always remote.